lib/mixlib/install/options.rb

```diff
@@ -155,6 +160,16 @@ def resolve_platform_version_compatibility_mode!
         end
       end
 
+      def proxy_settings
```

Comment:
mixlib install will not try to read environment variables. They must be explicitly set. It is not intended to work like other data transfer tools (curl) in that manner. Plus, this WIP only includes the proxy settings for using the API. It will get more complex as we add the proxy config for the install scripts.